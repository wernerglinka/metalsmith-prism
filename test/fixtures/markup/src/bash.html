<code class="language-bash">
$ echo "Hello World"
</code>
