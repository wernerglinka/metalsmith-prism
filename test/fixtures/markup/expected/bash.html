<html><head></head><body><code class="language-bash">
  $ <span class="token builtin class-name">echo</span> <span class="token string">"Hello World"</span>
</code>
</body></html>
